sbin/pfdhcplistener

```diff
@@ -169,7 +169,7 @@ sub dhcp_detector {
     # updating process name so we know what interface we are listening on
     # WARNING: the format is expected by watchdog in pf::services. Don't change lightly.
     $PROGRAM_NAME = basename($PROGRAM_NAME) . ": listening on $interface";
-    $pcap = Net::Pcap::pcap_open_live( $interface, 576, 1, 0, \$err );
+    $pcap = Net::Pcap::pcap_open_live( $interface, $Config{advanced}{pfdhcplistener_packet_size} || 576, 1, 0, \$err );
```

Comment:
The was more of a check to make sure that someone did not put 0.
But you are right we should just take it from pf.conf.defaults.
I will add a check in checkup.pm to not allow a size less than 576